DOC-821 Rejection and Return reasons for Merchant payment methods

ntombing opened the pull request.

max-koro commented on docs/topics/merchants/online/cards/index.mdx:
```diff
@@ -135,11 +135,35 @@ import DayTimeStandard from '../../../partials/_day-time.mdx';
+| `SoftDecline3dsExpiration` | The 3D Secure authentication session expired before completion. |
+| `SoftDecline3dsFailure` | The 3D Secure authentication failed. |
+| `SoftDecline3dsNotSupported` | The card does not support 3D Secure authentication. |
+| `SoftDecline3dsRequired` | The transaction requires 3D Secure authentication, but it was not completed. |
```
```suggestion
| `SoftDecline3dsRequired` | 3D Secure authentication was required but not completed. |
```

max-koro commented on docs/topics/merchants/online/cards/index.mdx:
```diff
@@ -135,11 +135,35 @@ import DayTimeStandard from '../../../partials/_day-time.mdx';
+| `SoftDeclineCardExpired` | The card used is no longer valid because it has expired. |
+| `SoftDeclineCardNotActivated` | The card has not been activated yet. The card holder should reach out to their card issuer for more information. |
+| `SoftDeclineCardNotSupported` | The card is not supported for this type of transaction. The card holder should reach out to their card issuer for more information. |
+| `SoftDeclineCvcCheckFailed` | The CVC security check did not succeed. |
```
```suggestion
| `SoftDeclineCvcCheckFailed` | The card security code (CVC) verification failed. |
```

max-koro reviewed (changes requested):
This looks fantastic. Really thorough work!

Just a couple of small polish items:
- I noticed we use both "rejected" and "declined" - should we pick one and stick with it?
- Quick style thing: could we swap `reach out to` for `contact`? Reads a bit smoother
- Can we make instances of `The transaction was refused by Swan for internal reasons.` more descriptive to minimize Support tickets? e.g. `Swan declined the transaction due to internal risk management or compliance requirements.`

ntombing commented:
I updated where relevant, but here I chose to:

> I noticed we use both "rejected" and "declined" - should we pick one and stick with it?

Not sure I 100% agree. I get why; the table is about rejections/returns. 
However, since the code or API uses `decline`, I opted to keep the term "declined" in the definition for alignment with the technical identifier. I did the same thing for `SwanRefused` - the payment is rejected because Swan refused...

> Quick style thing: could we swap reach out to for contact? Reads a bit smoother

Updated this to contact. 

`Can we make instances of The transaction was refused by Swan for internal reasons. more descriptive to minimize Support tickets? e.g. Swan declined the transaction due to internal risk management or compliance requirements.
`
Yes, this has been updated.

max-koro reviewed (approved):
LGTM! :D

ntombing commented on docs/topics/merchants/online/sdd/index.mdx:
```diff
@@ -147,26 +147,49 @@ Returns and refunds are only possible with **Core** and can be requested up to 8
+
+| Rejection reason | Explanation |
+|---|---|
+| `BankRefused` | The payment was refused by the debtor's bank.. |
```
```suggestion
| `BankRefused` | The payment was refused by the debtor's bank. |
```
